lib/app/views/onc_program.erb

```diff
@@ -648,6 +648,27 @@
               value: instance.bulk_url,
               })%> 
 
+
+        <div class="form-group"
+          data-requiredby="<%= test_set.variable_required_by(:attachment_requires_token).map {|seq| seq.sequence_name}.join(',')%>"
+          data-prerequisite="attachment_requires_token"
+          id="op_attachment_requires_token"
+          >
+          <label style="display:block">Attachment URL Authorization
+            <small>Select SMART Bearer token if attachments are served from a location that accepts the same bearer token used for the SIngle Patient API.  
+            Select "Self-signed URL" if the attachments encode authorization information within the URL and bearer tokens are not required.  
+            Systems that use "Self-signed URLs" are expected to secure them in a manner consistent with SMART Bearer tokens.</small>
```

Comment:
> are expected to secure them in a manner consistent with SMART Bearer tokens.

What does this mean? Do we have a real requirement we could reference?

Reply:
This is derived from discussion of bulk data's requiresAccessToken element (see discussion [here](https://chat.fhir.org/#narrow/stream/179250-bulk-data/topic/301.20redirect)). The idea is that sever can tell client the attacment url is a regular url requires Bearer token or is a self-signed urls without Bearer token. Bulk Data v2 mentioned (similar requirement)[https://hl7.org/fhir/uv/bulkdata/export.html#attachments]

Reply:
It isn't clear to me from that discussion what "secure them in a manner consistent with SMART Bearer tokens" actually means.

Reply:
My understanding is the same way server can handle access token, like server can revoke, expire and renew an access token. Should (or could) server do the same on an self-signed url?

Reply:
The actual expectation for self-signed url is still up for discussion as mentioned above. So we mentioned there is an option to use self-signed url *but Inferno Program testing does NOT allow that option at this time*